eng/pipelines/templates/scripts/replace-relative-links.yml

```diff
@@ -208,3 +208,7 @@ steps:
                 except Exception as e:
                     logging.error(e)
                     exit(1)
+
+  - script: |
+        git diff
```

Comment:
To help with some of the noise I'd suggest add the `-U0` option which tells it to not add context lines before and after the diff. That should help only show the real changes and not a bunch of other text that isn't easy to see why it is there.

Reply:
Sounds sweet. Will do so.